includes/functions/metadata.php

```diff
@@ -193,168 +237,210 @@ function resource_data_init() {
 	$prefix = 'lc_resource_';
 
 	$cmb = new_cmb2_box(
-		array(
+		[
 			'id'           => 'resource_data',
 			'title'        => pll__( 'Resource Data' ),
-			'object_types' => array( 'lc_resource' ),
+			'object_types' => [ 'lc_resource' ],
 			'context'      => 'normal',
 			'priority'     => 'high',
 			'show_names'   => true,
-		)
+		]
 	);
 
 	$cmb->add_field(
-		array(
+		[
 			'name'        => pll__( 'Permanent Link' ),
 			'description' => pll__( 'A permanent link to the resource.' ),
 			'id'          => $prefix . 'permanent_link',
 			'type'        => 'text_url',
-			'protocols'   => array( 'http', 'https' ),
+			'protocols'   => [ 'http', 'https' ],
+			'attributes'  => [
+				'data-validation' => 'true',
+				'data-required'   => 'true',
+			],
+		]
+	);
+
+	// TODO: Validate that the URL starts with https://perma.cc/
+	$cmb->add_field(
+		[
+			'name'        => pll__( 'Perma.cc Link', 'learning-commons-framework' ),
+			'description' => pll__( 'A link or links to an archival copy of the resource on <a href="https://perma.cc">Perma.cc</a>. If the resource spans multiple pages on Perma.cc, you may add multiple links.', 'learning-commons-framework' ),
+			'id'          => $prefix . 'perma_cc_links',
+			'type'        => 'text_url',
+			'repeatable'  => true,
+			'protocols'   => [ 'http', 'https' ],
+			'text'        => [
+				'add_row_text' => pll__( 'Add Link' ),
+			],
+			'attributes'  => [
+				'aria-label'      => pll__( 'Perma.cc Link' ),
+				'data-validation' => 'true',
+				'data-domain'     => 'perma.cc',
+			],
+		]
+	);
+
+	// TODO: Validate that the URL starts with https://web.archive.org/
+	$cmb->add_field(
+		[
+			'name'        => pll__( 'Wayback Machine Link', 'learning-commons-framework' ),
+			'description' => pll__( 'A link or links to an archival copy of the resource on the <a href="https://web.archive.org">Wayback Machine</a>. If the resource spans multiple pages on the Wayback Machine, you may add multiple links.', 'learning-commons-framework' ),
+			'id'          => $prefix . 'wayback_machine_links',
+			'type'        => 'text_url',
+			'repeatable'  => true,
+			'protocols'   => [ 'http', 'https' ],
```

Comment:
@jhung Added http protocol support in https://github.com/platform-coop-toolkit/learning-commons-framework/pull/84/commits/4857c36a10dfea441ee2d3ab35604cf81cb3ee31.